src/globus_cli/commands/timer/create/transfer.py

```diff
@@ -193,26 +193,21 @@ def transfer_command(
         )
         click.echo(click.style(msg, fg="yellow"), err=True)
 
-    if recursive is not None:
-        if batch:
-            # avoid 'mutex_option_group', emit a custom error message
-            option_name = "--recursive" if recursive else "--no-recursive"
-            raise click.UsageError(
-                f"You cannot use `{option_name}` in addition to `--batch`. "
-                f"Instead, use `{option_name}` on lines of `--batch` input which "
-                "need it."
-            )
-
-        if delete and not recursive:
-            raise click.UsageError(
-                "The `--delete` option cannot be specified with `--no-recursive`."
-            )
-        if delete_destination_extra and not recursive:
-            raise click.UsageError(
-                "The `--delete-destination-extra` option cannot be specified with "
-                "`--no-recursive`."
-            )
-
+    # avoid 'mutex_option_group', emit a custom error message
+    if recursive is not None and batch:
+        option_name = "--recursive" if recursive else "--no-recursive"
+        raise click.UsageError(
+            f"You cannot use `{option_name}` in addition to `--batch`. "
+            f"Instead, use `{option_name}` on lines of `--batch` input which "
+            "need it."
```

Comment:
Elsewhere, these lines are joined. Perhaps an IDE setting auto-wrapped this?

Not critical to fix, but I wanted to bring it up in case there's something that auto-wrapped this unnecessarily (our linters don't currently un-wrap text when it could fit on a single line).

```suggestion
            f"Instead, use `{option_name}` on lines of `--batch` input which need it."
```